media-libs/opensubdiv/opensubdiv-3.4.0.recipe

```diff
@@ -92,35 +86,35 @@ defineDebugInfoPackage opensubdiv$secondaryArchSuffix \
 
 BUILD()
 {
-	mkdir -p build
-	cd build
-
-	export GLEW_LOCATION="$portPackageLinksDir/devel~libglew/$relativeDevelopDir/"
-	export GLEW_INC_LOCATION="$portPackageLinksDir/devel~libglew/$relativeIncludeDir/"
+	mkdir -p build && cd build
 
-	cmake .. \
+	cmake .. -DCMAKE_BUILD_TYPE=Release \
+		-DCMAKE_INSTALL_PREFIX=$prefix \
+		-DCMAKE_BINDIR_BASE=$prefix/bin \
+		-DCMAKE_INCDIR_BASE=$includeDir \
+		-DCMAKE_LIBDIR_BASE=$libDir \
 		-DNO_EXAMPLES=ON \
 		-DNO_TUTORIALS=ON \
-		$cmakeDirArgs
+		-DNO_GLFW_X11=ON \
```

Comment:
Oh, i see.

Reply:
It actually failed on not finding X11 for this